app/bg_loops.py

```diff
@@ -42,7 +42,8 @@ async def _remove_expired_donation_privileges(interval: int) -> None:
         expired_donors = await app.state.services.database.fetch_all(
             "SELECT id FROM users "
             "WHERE donor_end <= UNIX_TIMESTAMP() "
-            "AND priv & 48",  # 48 = Supporter | Premium
+            "AND priv & :donor_priv",  # 48 = Supporter | Premium
+            {"donor_priv": Privileges.DONATOR.value},
```

Comment:
does this work without `.value`? i think it should if Privileges is wrapped with the pymysql_encode util

Reply:
also i'd prefer this be explicit
```suggestion
            {"donor_priv": Privileges.SUPPORTER | Privileges.PREMIUM},
```